mcdb.spec

```diff
@@ -1,7 +1,7 @@
-### RPM external mcdb 1.0.2
+### RPM external mcdb 1.0.3
 Source: http://mcdb.cern.ch/distribution/api/%{n}-api-%{realversion}.tar.gz
 Requires: xerces-c
-Patch0: mcdb-1.0.2-gcc45
+#Patch0: mcdb-1.0.2-gcc45
```

Comment:
ijaz is doing the cleaning up for stable branch now and will merge it for 11h IB